rs_cleaner.py

```diff
@@ -0,0 +1,24 @@
+class RSCleaner:
+
+    def __init__(self):
+        self.transaction_types = {'ins', 'inb', 'nic', 'nis'}
+
+    def clean_timestamp(self, ts_string):
+        """
+        will return two timestamps as strings. The first is the timestamp of the original post. The second,
+        if it exists, will be the timestamp of the edit.
```

Comment:
the comment should indicate that it may return `None` for the second timestamp if the post wasn't edited